packages/react-data-grid/src/HeaderRow.tsx

```diff
@@ -44,6 +44,10 @@ export default class HeaderRow extends React.Component<Props> {
     );
   }
 
+  getHeaderCellKey(column: Column): string {
+    return `headercell-${column.key}`;
```

Comment:
I think we can removed `header-cell` and just use `column.key`.  

https://reactjs.org/docs/lists-and-keys.html#keys-must-only-be-unique-among-siblings